liferay-lima/liferay-portal
# LPS-165648 Create simple repository browser tag

# What is this about?

This is a tag that offers a limited interface to browse and operate on
repository documents and folders.

![image](https://user-images.githubusercontent.com/729897/195862023-61f5bf39-8463-4469-8db4-8784f38b9421.png)

# How do I test this?

You'll need to add the tag to a portlet JSP (ideally, one with no other content):
```
<liferay-document-library:repository-browser />
```

You can specify a particular repository to browse with the `repositoryId` attribute:
```
<liferay-document-library:repository-browser repositoryId="<%= repositoryId %>" />
```

After that, you should be able to:

1. Navigate through the folder structure,
2. Delete documents, shortcuts and folders,
3. Rename documents, folders and shortcuts,
4. Upload a single document,
5. Create a folder,
6. Search entries by name.
